__tests__/app.test.js

```diff
@@ -152,15 +152,96 @@ describe("app", () => {
         .get("/api/articles/invalid/comments")
         .expect(400)
         .then(({ body }) => {
-          expect(body).toEqual({ msg: "Invalid ID" });
+          expect(body).toEqual({ msg: "Bad Request" });
         });
     });
     test("404: responds with article not found if valid but non-existent article id entered", () => {
       return request(app)
         .get("/api/articles/999999/comments")
         .expect(404)
-        .then(({ body }) => {
-          expect(body).toEqual({ msg: "Article does not exist" });
+        .then((response) => {
+          expect(response.body.msg).toEqual("Not Found");
+        });
+    });
+  });
+
+  describe("POST /api/articles/:article_id/comments", () => {
+    test("201: Should accept username and body keys, and respond with an array containing the posted comment", () => {
+      const addedComment = {
+        username: "icellusedkars",
+        body: "I hate streaming noses",
+      };
+
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(addedComment)
+        .expect(201)
+        .then((result) => {
+          expect(result.body.comments).toBeInstanceOf(Object);
+          expect(result.body.comments).toHaveProperty("comment_id", 19);
+          expect(result.body.comments).toHaveProperty(
+            "body",
+            "I hate streaming noses"
+          );
+          expect(result.body.comments).toHaveProperty("article_id", 1);
+          expect(result.body.comments).toHaveProperty("author");
+          expect(result.body.comments).toHaveProperty("votes", 0);
+          expect(result.body.comments).toHaveProperty("created_at");
+        });
+    });
```

Comment:
Great test Joravar, well done. Good decision to check the actual values on the response body also to gain extra confidence that your POST request worked.